docs/README-Extending.md

```diff
@@ -209,7 +259,7 @@ if (zoweExplorerApi) {
   // the user does not have the CICS CLI Plugin installed and profiles created, yet.
   const meta = await CoreUtils.getProfileMeta();
   await zoweExplorerApi.getExplorerExtenderApi().initForZowe("cics", meta);
-
+}
```

Comment:
@traeok and shouldn't this closing bracket be somewhere after `const firstProfile = profilesCache.loadNamedProfile(profileNames[0]);`